academic/import_bibtex.py

```diff
@@ -15,21 +16,37 @@
 from academic import utils
 from academic.editFM import EditableFM
 
+
 # Map BibTeX to Academic publication types.
+class PublicationType(Enum):
```

Comment:
Since you're proposing a class is created for pup. types, please consider moving `PublicationType` class and `PUB_TYPES` const to a new file (and adjust the imports). Thanks.